stories/s1grdew.stories.js

```diff
@@ -216,8 +217,44 @@ export const getMapProcessingFromLayer = () => {
   return wrapperEl;
 };
 
-export const findTiles = () => {
-  const layer = new S1GRDAWSEULayer({ instanceId, layerId });
+export const findTilesSearchIndex = () => {
+  const layer = new S1GRDAWSEULayer({
+    instanceId,
+    layerId,
+    acquisitionMode: AcquisitionMode.EW,
+    polarization: Polarization.DH,
+    resolution: Resolution.MEDIUM,
+  });
+  const containerEl = document.createElement('pre');
+
+  const wrapperEl = document.createElement('div');
+  wrapperEl.innerHTML = '<h2>findTiles</h2>';
+  wrapperEl.insertAdjacentElement('beforeend', containerEl);
+
+  const perform = async () => {
+    setAuthToken(null);
+    const data = await layer.findTiles(
+      bbox4326,
+      new Date(Date.UTC(2020, 2 - 1, 2, 0, 0, 0)),
+      new Date(Date.UTC(2020, 2 - 1, 2, 23, 59, 59)),
+      5,
+      0,
+    );
+    renderTilesList(containerEl, data.tiles);
+  };
+  perform().then(() => {});
+
+  return wrapperEl;
+};
+
+export const findTilesSearchCatalog = () => {
```

Comment:
I didn't investigate why, but this story issues a `searchIndex` request instead of `catalog`. The same happens with S-1 IW.